config/initializers/sentry.rb

```diff
@@ -1,13 +1,12 @@
 ##
 # * https://sentry.io/for/rails/
-# * https://github.com/getsentry/raven-ruby
+# * https://github.com/getsentry/sentry-ruby
 #
-Raven.configure do |config|
+Sentry.configure do |config|
   # Sentry will be enabled if SENTRY_DSN exists. Sentry reporting will work if
   # SENTRY_DSN has a meaningful value.
   config.dsn = ENV["SENTRY_DSN"]
 
-  # We expect SENTRY_ENV to always be set so fail with an error if it is
-  # missing
+  # Set Sentry environment to be current environment if SENTRY_ENV is not set
```

Comment:
🐘 putting this here for possible future readers (including myself 😅)

The backstory for it is that `Rails.env` is typically fine to use except some platforms (aka Heroku) very strongly believe that you should only have `development`, `test`, and `production` for your env values (e.g. no `staging` or `uat`) - and that instead of checks like `Rails.env.staging?` you should be using feature flags / env variables to toggle specific behaviour.

Hence, we check for `SENTRY_ENV` first to allow us to provide a more specific environment name without affecting the environment mode the app is actually running in since that could be impacted by the hosting service being used.